cli/internal/context/context.go

```diff
@@ -22,7 +22,7 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
-const GLOBAL_CACHE_KEY = "Ba weep granna weep ninny bong"
+const _globalCacheKey = "Real G's move in silence like lasagna"
```

Comment:
🤣

Reply:
Love it

Reply:
[Mom's Spaghetti](https://www.youtube.com/watch?v=SW-BU6keEUw)